packages/terra-consumer-nav/src/components/user-profile/UserProfile.jsx

```diff
@@ -0,0 +1,166 @@
+    };
+
+    this.openPopup = this.openPopup.bind(this);
+    this.closePopup = this.closePopup.bind(this);
+  }
+
+  openPopup() {
+    this.setState({ isOpen: true });
+  }
+
+  closePopup() {
+    this.setState({ isOpen: false });
+  }
+
+  render() {
+    const { profile, ...customProps } = this.props;
+    const profileLinks = profile.links.map((item) => {
+      const currentItem = (profile.links.length > 0 &&
+      <a href={item.uri} onClick={() => window.location = item.uri} className={cx('profile-link')} style={{ padding: '0px' }}>
+        <Arrange
+          align="stretch"
+          fitStart={<div className={cx('profile-icon')}>{item.icon}</div>}
+          fill={<div style={{ paddingLeft: '10px' }}>{item.text}</div>}
+        />
+      </a>
+      );
+
+      return (
+        <div key={`${item.text}`} className={cx('divider-bottom')} style={{ padding: '20px' }}>
+          {currentItem}
+        </div>);
+    });
+
+    const defaultElement = (
+      <Overlay isOpen={this.state.isOpen} backgroundStyle="dark" isScrollable>
+        <div className={cx('profile-modal-header')}>
+          <Arrange
+            fitStart={<div className={cx('profile-modal-title')}>{translations.settings}</div>}
+            fitEnd={<button className={cx('modal-close-button')} onClick={() => this.closePopup()}>{<div className={cx('modal-close-icon')}><CloseIcon /></div>}</button>}
+            align="center"
+          />
+        </div>
+        <div className={cx('profile-modal')}>
+          {(<div>
+            {profileLinks}
+            <div style={{ padding: '20px' }}>
+              <a className={cx('profile-link')} href={profile.signoutUrl} onClick={() => window.location = profile.signoutUrl} style={{ padding: '0px' }}>
+                {translations.help}
+              </a>
+            </div>
+            <div className={cx('divider-top')} style={{ padding: '20px' }}>
+              <a
+                className={cx('profile-link')}
+                href={profile.signoutUrl}
+                onClick={() => window.location = profile.signoutUrl}
+                style={{ padding: '0px' }}
+              >
+                {translations.signout}
+              </a>
+            </div>
+          </div>)}
+        </div>
+      </Overlay>);
+
+    const large = (
+      <Popup
+        isOpen={this.state.isOpen}
+        onRequestClose={this.closePopup}
+        targetRef={() => document.getElementById('profile-link-icon')}
+        isArrowDisplayed
+        contentWidth="320"
+        contentHeight="400"
+        contentAttachment="top right"
+      >
+        {<div>
+          {profileLinks}
+          <div style={{ padding: '20px' }}>
+            <a className={cx('profile-link')} href={profile.signoutUrl} onClick={() => window.location = profile.signoutUrl} style={{ padding: '0px' }}>
+              {translations.help}
+            </a>
+          </div>
+          <div className={cx('divider-top')} style={{ padding: '20px' }}>
```

Comment:
Removed all the inline styles. There is one to add sample content in the main container of the Layout.jsx. This is used to test the help button position for the different case.The layout is now part of the test folder, so kept it for the test, can remove it if suggested.